src/components/InstructorPage.js

```diff
@@ -0,0 +1,257 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router-dom'
+
+import Button from '@material-ui/core/Button'
+
+import './InstructorPage.css'
+
+//Services
+import peerReviewService from '../services/peerReview'
+
+//Actions
```

Comment:
If this is not necessary is should be just removed

Reply:
Removed the unused action from the comment.